__tests__/app.test.js

```diff
@@ -44,30 +45,100 @@ describe('appTests', () => {
             return request(app)
-                .get('/api/notanexistingendpoint')
-                .expect(404)
-                .then(({body}) => {
-                    console.log(body.msg)
-                    console.log(body, "<<<<<")
-                    expect(body.msg).toBe('Not Found')
+                .get('/api/reviews')
+                .expect(200)
+                .then(({body})=>{
+                    
+                    expect(Array.isArray(body.reviews)).toBe(true)
+                })
+        })
+        test('returns an array of review objects', () => {
+            return request(app)
+                .get('/api/reviews')
+                .expect(200)
+                .then(({body})=>{
+                    expect(Object.prototype.toString.call(body.reviews[0])).toBe('[object Object]')
+                })
+        })
+        test('should have 9 length', () => {
+            return request(app)
+                .get('/api/reviews')
+                .expect(200)
+                .then(({body})=>{
+                    expect(Object.keys(body.reviews[0]).length).toBe(9)
+                })
+        })
+        test('objects should have correct properties', () => {
+            return request(app)
+                .get("/api/reviews")
+                .expect(200)
+                .then(({body})=>{
+                    body.reviews.forEach(review => {
+                        expect(review).toHaveProperty("owner", expect.any(String))
+                        expect(review).toHaveProperty("title", expect.any(String))
+                        expect(review).toHaveProperty("review_id", expect.any(Number))
+                        expect(review).toHaveProperty("category", expect.any(String))
+                        expect(review).toHaveProperty("review_img_url", expect.any(String))
+                        expect(review).toHaveProperty("created_at", expect.any(String))
+                        expect(review).toHaveProperty("votes", expect.any(Number))
+                        expect(review).toHaveProperty("designer", expect.any(String))
+                        expect(review).toHaveProperty("comment_count", expect.any(Number))
+                    });
+                })
+        })
+        test('return objects should be sorted by date in descending order', () => {
+            return request(app)
+                .get('/api/reviews')
+                .expect(200)
+                .then(({body})=>{
+                    const reviewDates = body.reviews.map((review)=>{
+                        const date = review.created_at.substring(0,4) + review.created_at.substring(5,7) + review.created_at.substring(8,10)
+                    
+                        return parseInt(date,10)
+                    });
+
+                    for (let i = 1; i < reviewDates.length; i++) {
+                        expect(reviewDates[i - 1] >= reviewDates[i]).toBe(true)
+                    }
                 })
+
```

Comment:
You can group all of these tests into one. Really we should only be doing a new test when we are requesting a new endpoint. If you are request the same end point multiple times to do multiple assertions, then just do one request and put all your assertions in that test